libwallet/src/internal/selection.rs

```diff
@@ -535,13 +535,13 @@ where
 	K: Keychain + 'a,
 {
 	// first find all eligible outputs based on number of confirmations
-	let mut eligible = wallet
-		.iter()
+	let mut eligible: Vec<OutputData> = wallet
+		.get_outputs_eligible()
```

Comment:
If you decide to change the name of the function, you need to change it here too